thepracticaldev/dev.to
# [ModCenter v1] Fix bugs post-launch

## What type of PR is this? (check all applicable)


## Description
After launching ModCenter v1, we encountered the following bugs:
- Article titles were blacked-out in dark themes
- Articles without tags showed a solitary `#` mark
- Articles with long titles wrapped around (per [ModCenter specs](https://www.figma.com/file/Jgq2Qg3MqmjQ4uVdJtyAsS/Moderation?node-id=419%3A6974), long titles should be truncated)

This PR fixes these bugs.


## QA Instructions, Screenshots, Recordings

Fire up your local env, navigate to `/mod` and ensure you have some articles with (1) long titles (2) no tags

<img width="1424" alt="Screen Shot 2020-06-24 at 9 27 54 AM" src="https://user-images.githubusercontent.com/32520970/85565285-009f1380-b5fd-11ea-9427-7fc4d6e0a51f.png">


## Added tests?


## Added to documentation?



## [optional] What gif best describes this PR or how it makes you feel?

![Cartoon fixing car](https://media.giphy.com/media/daxVu1mZErt2qKYdtX/giphy.gif)
